client/src/components/Card.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import colors from '../utils/colors';
+
+const Container = styled.article`
+  display: flex;
+  flex-flow: column nowrap;
+  width: 330px;
+  height: 350px;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.4);
+`;
+const Image = styled.img`
+  display: flex;
+  object-fit: cover;
+  height: 243px;
+  padding: 0;
+  border-radius: 5px 5px 0 0;
+`;
+
+const InfoBox = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: center;
+  height: 107px;
+  border-radius: 0 0 5px 5px;
+  background-color: ${colors.bgSecondary};
+`;
+
+const Title = styled.h1`
+  font-family: SF Pro Rounded Medium;
```

Comment:
This font can be added to your theme. :-)

Reply:
What exactly do you mean? ) I use a lot of fonts in my App...  Is there another way to get a better structure? 

<img width="509" alt="Screenshot 2020-04-17 at 16 55 07" src="https://user-images.githubusercontent.com/27225956/79582653-540c5700-80cc-11ea-8628-6bf874605985.png">